portal-web/test/functional/com/liferay/portalweb/functions/Uncheck.function

```diff
@@ -87,4 +87,22 @@ definition {
 		}
 	}
 
+	function uncheckVisible {
```

Comment:
I think this function should be renamed to uncheckNotVisible, like the similar function Check.checkNotVisible.

Besides this, everything else looks good to me. This can be forwarded after change the function name.

Reply:
hmmm I was under the assumption that Check.checkNotVisible meant to check if there wasn't a visible mark in the checkbox. So that's why I did Uncheck.uncheckVisible, uncheck the visible mark in the checkbox. What do you think @john-co?

Reply:
Agree with @zhangyan0701. `.uncheckNotVisible` follows the pattern from Check.function 

We're using selenium.waitForElementPresent here so only asserting the element is present in the DOM. We're not asserting that the element is necessarily "visible" or appears on the page, hence uncheck**Not**Visible. Side note: The checkbox does appear on the page, but for some reason selenium sometimes doesn't see it as "visible"...

On the other hand, if we look at the default Check, it uses selenium.waitForVisible so it's more like `.checkVisible`

Reply:
Thanks @john-co  for the explanation!